research/src/sources/fabric.json5

```diff
@@ -361,7 +361,34 @@
     },
     {
       "name": "Shimmer",
-      "url": "https://developer.microsoft.com/en-us/fabric#/controls/web/shimmer"
+      "openUIName": "Skeleton",
+      "url": "https://developer.microsoft.com/en-us/fabric#/controls/web/shimmer",
+      "concepts": [
+        {
+          "name": "default",
+          "image": "fabric-shimmer-basic.png"
+        },
+        {
+          "name": "default",
+          "image": "fabric-shimmer-basic-elements.png"
+        },
+        {
+          "name": "variations",
+          "image": "fabric-shimmer-basic-variations.png"
```

Comment:
this one is rendering broken due to being undefined:

`<img alt="An image of the variations concept on a Skeleton component in UI Fabric." srcset="undefined 2x" title="fabric-shimmer-basic-variations.png" style="display: inline-block; vertical-align: middle; margin: 0px;">`

Reply:
Good catch @gregwhitworth . Fixed